cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxVideoView.java

```diff
@@ -307,7 +307,11 @@ private void openVideo() {
             mDuration = -1;
             mCurrentBufferPercentage = 0;
             if (mIsAssetRouse) {
-                AssetFileDescriptor afd = mCocos2dxActivity.getAssets().openFd(mVideoFilePath);
+                AssetFileDescriptor afd = null;
+                if (Cocos2dxHelper.getObbFile() != null) {
+                    afd = Cocos2dxHelper.getObbFile() .getAssetFileDescriptor(mVideoFilePath);
+                }
+                if(afd == null) afd = mCocos2dxActivity.getAssets().openFd(mVideoFilePath);
```

Comment:
Why check OBB file first? Is using OBB file more common?

Reply:
It doesn't matter what order check.

Reply:
But check assets may make it more faster if video files are more commonly stored in assets.

Reply:
yes, you are right.

Reply:
@minggo I change the check order, please review